graylog2-server/src/main/java/org/graylog2/indexer/results/ScrollResult.java

```diff
@@ -41,12 +41,13 @@
     };
 
     public interface Factory {
-        ScrollResult create(SearchResult initialResult, String query, List<String> fields);
+        ScrollResult create(SearchResult initialResult, @Assisted("query") String query, @Assisted("scroll") String scroll, List<String> fields);
```

Comment:
Is it possible to keep the old factory method and simply mark it as `@Deprecated`?
Otherwise third party plugins might break due to this change.